sqlmesh/core/engine_adapter/redshift.py

```diff
@@ -136,6 +137,17 @@ def _build_create_table_exp(
                 for target in plan["targetlist"]:  # type: ignore
                     if target["name"] == "TARGETENTRY":
                         resdom = target["resdom"]
+                        resname = resdom["resname"]
+                        if resname == "<>":
+                            # A synthetic column added by Redshift to compute a window function.
+                            continue
+                        if resname == "? column ?":
```

Comment:
Does the plan add spaces between `?` and `column`? I don't remember seeing them but I could be wrong.

Reply:
Tokenizer does. Just like the `- 1` example below